python/cudf/cudf/core/series.py

```diff
@@ -791,7 +791,9 @@ def reindex(self, index=None, copy=True):
 """,
         )
     )
-    def reset_index(self, level=None, drop=False, name=None, inplace=False):
+    def reset_index(
+        self, level=None, drop=False, name=None, inplace=False
+    ):  # noqa: D102, E501
```

Comment:
Everywhere that we have `noqa: D102`, do those correspond to places where docstrings are inherited? If so, can `pydocstyle` not be configured to recognize that?

Reply:
Edit: I guess [not](https://github.com/PyCQA/pydocstyle/issues/309)